auth/src/main/java/com/firebase/ui/auth/ui/HelperActivityBase.java

```diff
@@ -70,4 +75,8 @@ public void startSaveCredentials(
                 this, getFlowParams(), credential, response);
         startActivityForResult(intent, RequestCodes.CRED_SAVE_FLOW);
     }
+
+    public void onMergeFailure(IdpResponse response) {
+        finish(ErrorCodes.ANONYMOUS_UPGRADE_MERGE_CONFLICT, response.toIntent());
```

Comment:
I'd prefer we not do this. We have this inlined in all the other call sites so we don't end up with a God activity.

Reply:
What do you suggest we do?

Reply:
Oh, sorry for not being clearer, I was just saying to inline this like we do elsewhere. This would be easiest: https://www.jetbrains.com/help/idea/inline.html